app/src/test/resources/structs.h

```diff
@@ -0,0 +1,23 @@
+//
+// Created by Michael C. on 8/30/22.
+//
+
+#ifndef BOTTOM_NAV_STRUCTS_H
+#define BOTTOM_NAV_STRUCTS_H
+
+
+// @canPayloadStruct CAN_PEDAL_POS = CanPedalPosStruct
+const uint16_t CAN_PEDAL_POS = 0x282;
+
+struct CanPedalPosStruct {
```

Comment:
fyi this actually won't fit within CAN, but it's fine as a test case
CAN payloads are 8 bytes max, this is 4+1+1+2+2+4+4=18